CONTRIBUTING.md

```diff
@@ -1009,7 +1015,9 @@ $ kind create cluster --name=dc1 && kind create cluster --name=dc2
   `-consul-k8s-image=<your-custom-image>` && `-consul-image=<your-custom-image>`
 * You can set custom helm flags by modifying the test file directly in the respective directory.
 
-Finally, run the test like shown above:
+Finally, you have two options on how you can run your test:
+1. Take the following steps, this will run the test through to completion but not teardown any resources created by the test so you can inspect the state of the cluster
+at that point. You will be responsible for cleaning up the resources or deleting the cluster entirely when you're done.
```

Comment:
I think we should probably call out the specific flags that make this true - from what I can see, most of the command below is common instruction, we're just offering an alternative to `-no-cleanup-on-failure`, right?

Reply:
we can call them out specifically, we have a list of the flags and what they do earlier in this doc as well https://github.com/hashicorp/consul-k8s/blob/main/CONTRIBUTING.md#pre-requisites

and yeah the `WaitForInput` is similar to `--no-cleanup-on-failure` but allows you to inspect the cluster at different points of the setup rather than the state it's in when the test fails

Reply:
Ok cool. For this doc change, my main thinking was this blurb implies now there's two divergent paths, but really, only the first one tells you what command to run.

So we should probably either have a single command without the flag and options 1 + 2 to use it or `WaitForInput`, or just have two similar commands, one per option. Does that sound right?

Reply:
that makes sense, let me update this to show how that would work

Reply:
updated the text a bit to show how the two different paths work

Reply:
Nice, LGTM, thank you!